met-web/src/components/engagement/new/view/EngagementDescription.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { Suspense } from 'react';
+import { BodyText, Header2 } from 'components/common/Typography';
+import { Link } from 'components/common/Navigation';
+import { Engagement } from 'models/engagement';
+import { Grid, Skeleton } from '@mui/material';
+import { colors } from 'components/common';
+import { Await, useLoaderData } from 'react-router-dom';
+import { Editor } from 'react-draft-wysiwyg';
+import { getEditorStateFromRaw } from 'components/common/RichTextEditor/utils';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeftLong } from '@fortawesome/pro-light-svg-icons';
+import { Widget } from 'models/widget';
+import { WidgetSwitch } from 'components/engagement/view/widgets/WidgetSwitch';
+
+export const EngagementDescription = ({ engagement }: { engagement: Engagement }) => {
+    const { widgets } = useLoaderData() as { widgets: Widget[] };
+    return (
+        <section id="cta-section">
```

Comment:
Could we label this accessibility region? In the last PR, we used an aria-label directly for something descriptive of the section. In this case, I think we can use the H2 below. You'd reference it with an `aria-labelledby` attribute

See here for the relevant technique: https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/region.html